test/models/ConferenceTest.scala

```diff
@@ -0,0 +1,151 @@
+      assert(Conference.convertMarkdownToHTML(testingString + "\n=") == "<h1>"+testingString+"</h1>\n")
+      assert(Conference.convertMarkdownToHTML(testingString + "\n-") == "<h2>"+testingString+"</h2>\n")
+      // blockquote
+      assert(Conference.convertMarkdownToHTML("> "+testingString) == "<blockquote>\n<p>"+testingString+"</p>\n</blockquote>\n")
+      assert(Conference.convertMarkdownToHTML(">"+testingString) == "<blockquote>\n<p>"+testingString+"</p>\n</blockquote>\n")
+      // lists
+      assert(Conference.convertMarkdownToHTML("- "+testingString) == "<ul>\n<li>"+testingString+"</li>\n</ul>\n")
+      assert(Conference.convertMarkdownToHTML("+ "+testingString) == "<ul>\n<li>"+testingString+"</li>\n</ul>\n")
+      assert(Conference.convertMarkdownToHTML("* "+testingString) == "<ul>\n<li>"+testingString+"</li>\n</ul>\n")
+      assert(Conference.convertMarkdownToHTML("-"+testingString) == "<p>-"+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("+"+testingString) == "<p>+"+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("*"+testingString) == "<p>*"+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("- "+testingString+"\n- "+testingString)
+        == "<ul>\n<li>"+testingString+"</li>\n<li>"+testingString+"</li>\n</ul>\n")
+      assert(Conference.convertMarkdownToHTML("1. "+testingString) == "<ol>\n<li>"+testingString+"</li>\n</ol>\n")
+      assert(Conference.convertMarkdownToHTML("1."+testingString) == "<p>1."+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("2)"+testingString) == "<p>2)"+testingString+"</p>\n") // space needed
+      assert(Conference.convertMarkdownToHTML("1. "+testingString+"\n2. "+testingString)
+        == "<ol>\n<li>"+testingString+"</li>\n<li>"+testingString+"</li>\n</ol>\n")
+      // links
+      val urlTestingString : String = "http:"+testingString
+      assert(Conference.convertMarkdownToHTML("<"+urlTestingString+">")
+        == "<p><a href=\""+urlTestingString+"\">"+urlTestingString+"</a></p>\n")
+      assert(Conference.convertMarkdownToHTML("["+testingString+"]"+"("+urlTestingString+")")
+        == "<p><a href=\""+urlTestingString+"\">"+testingString+"</a></p>\n")
+      // code
+      assert(Conference.convertMarkdownToHTML("`"+testingString+"`") == "<p><code>"+testingString+"</code></p>\n")
+      assert(Conference.convertMarkdownToHTML("    "+testingString) == "<pre><code>"+testingString+"\n</code></pre>\n")
+      assert(Conference.convertMarkdownToHTML("```\n"+testingString+"\n```") == "<pre><code>"+testingString+"\n</code></pre>\n")
+    }
+  }
+
+  @Test
+  def testGetInfoAsHTML() : Unit = {
+    /*
+     * Mostly a placeholder for potential future changes, which need to be tested differently.
+     */
+    val r : Random = new Random()
+    var testingInfo : String = ""
```

Comment:
Testing this (empty string) would actually be an interesting corner case

Reply:
Actually empty strings are tested implicitly at two points. First in 
```
  def testConvertMarkdownToHTML() : Unit = {
    val r : Random = new Random()
    // test nulls throwing a NullPointerException
    intercept[NullPointerException] {
      Conference.convertMarkdownToHTML(null)
    }
    // test empty markdown returning empty html
    assert(Conference.convertMarkdownToHTML("") == "")
```
, which is currently equivalent to the instance functions.
And a second time as part of the random testing since strings of length zero are allowed.
I, however, added the empty string corner case explicitly to the instance function tests for the sake of completeness.